service/matching/taskListManager_test.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/pborman/uuid"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
```

Comment:
this is most probably because of Go 1.21 != CI (1.20)

Reply:
Lets address this after @Groxx PR. Then everything will be unified.